### DISCUSSION TITLE
Is Disabling Preflight equivalent to deleting the @tailwind base; line?

### DISCUSSION THREAD

**Discussion by fnick851** [tailwindlabs/tailwindcss#6377] [Help] [answered]
Disabling Preflight:
```js
  // tailwind.config.js
  module.exports = {
    corePlugins: {
     preflight: false,
    }
  }
```

Or, remove the base CSS line:
```css
/* @tailwind base; */
```

Are these two completely equivalent?

**Comment by acalvino4** [comment, marked as answer]
Not completely
From [Functions and Directives](https://tailwindcss.com/docs/functions-and-directives) -
```
/**
 * This injects Tailwind's base styles and any base styles registered by
 * plugins.
 */
@tailwind base;
```

From [Preflight](https://tailwindcss.com/docs/preflight)
```
@tailwind base; /* Preflight will be injected here */
```

To summarize, the `@tailwind base` directive provides a layer for both tailwind and your own css to load code. Preflight is just _one_ thing (albeit probably the most significant) that gets loaded there.

**Reply by fnick851** [reply]
Thanks!
